products.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Sandwich Collective</title>
+    <link rel="stylesheet" href="src/main.css">
+    <link rel="stylesheet" href="src/products.css">
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Tajawal:400,700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>
+            The Sandwich Collective
+        </h1>
+            <img src="assets/shop-image.png" alt="small shop image" class="logo-image">
+    </header>
+    
+    <main>
+        <section id="category-section">
+            Sort your sandwiches: 
+            <button value="all" id="show-all-category">Show All</button>
+            <button value="hot" id="hot-category">Hot</button>
+            <button value="cold" id="cold-category">Cold</button>
+            <button value="breakfast" id="breakfast-category">Breakfast</button>
+        </section>
+
+        <section id="products-section">
+            <ul id="product-list"></ul>
+        </section>
+    </main>
+
+    <script src="/src/data/sandwiches.js" type="module"></script>
```

Comment:
this script tag for the data can be deleted - you are importing the sandwich data in your js file